poetry.lock

```diff
@@ -148,10 +148,10 @@ description = "The AWS SDK for Python"
 name = "boto3"
 optional = false
 python-versions = "*"
-version = "1.13.19"
+version = "1.13.15"
```

Comment:
Downgrade here. Please check all versions of the packages.

Reply:
Done